ImageViewer/Source/ImageFadeInHandler.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  ImageFadeInHandler.swift
+//  ImageViewer
+//
+//  Created by Kristian Angyal on 02/03/2016.
+//  Copyright © 2016 MailOnline. All rights reserved.
+//
+
+import Foundation
+
+final class ImageFadeInHandler {
+    
+    var presentedImages:[Int] = []
+    
+    func imagePresentedAtIndex(index: Int) {
+        
```

Comment:
I just couldn't come up with a good name at that time..so I was hoping you will have some suggestion, cool.